v2/concordium/types.proto

```diff
@@ -1925,6 +1955,8 @@ message BakerStakeThreshold {
 // Root updates are the highest kind of key updates. They can update every other set of keys,
 // even themselves. They can only be performed by Root level keys.
 message RootUpdate {
+  // This field will not be extended in future versions of the API, instead new data versions will
+  // be using optional fields next to this field.
```

Comment:
I think it's more accurate to say that e.g. `AuthorizationsV1` will be extended with optional fields.